### PR TITLE
New Port: Crysalis

### PR DESCRIPTION
**Pull request by cdeletre**
# New Port for Crysalis

## Game Information
- **Title**: Crysalis
- **URL**: https://www.rpgmakerweb.com/additional-downloads#sample-games

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure

```
.
├── Crysalis.sh
├── README.md
├── crysalis
│   ├── LICENSE_falcon-mkxp
│   ├── LICENSE_music.txt
│   ├── LICENSE_rpgmaker_RTP.txt
│   ├── PLACE_GAME_FILES_HERE.TXT
│   ├── conf
│   ├── crysalis.gptk
│   ├── libs.aarch64
│   └── tools.aarch64
├── gameinfo.xml
├── port.json
└── screenshot.jpg
```
## Controls

| Button | Action |
|--|--| 
|A|Interact / Jump|
|B|Cancel / Back|
|Y|Bicycle|
|DPAD|Movement|
|L-STICK|Alt. Movement|
|R-STICK|Battle inputs|
|Back buttons|Alt. battle inputs|
|Start|Menu|
|Select|Interact|

## Additional informations

Copy *Crysalis.exe* (extract it from [crysalis.zip](https://www.rpgmakerweb.com/additional-downloads)) and *RPGVXAce_RTP.zip* ([RPG Maker VX Ace Run Time Package](https://www.rpgmakerweb.com/run-time-package)) in *ports/roms/crysalis*.

The TheLaggyPyro is the main porter for this game, I've just given him a hand with the scripting and the PR :)

### CODE REVIEW

**Review by kloptops — APPROVED** [review]
👍

File: ports/crysalis/port.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    ],
+    "desc": "Crysalis – the water of life. For generations, people lived their lives, blessed by the divine water. After finding out from the king that the town Ciuna’s Crysalis spring has run dry, Mio, a Crysalis researcher, is called in.\n\nGo on an epic journey with Mio, and friends. Beautiful maps, elegant music, and fun areas to explore!",
+    "desc_md": null,
+    "inst": "Copy Crysalis.exe (extract it from crysalis.zip) and RPGVXAce_RTP.zip in the roms/ports/crysalis folder.",
+    "inst_md": null,
+    "genres": [
+      "rpg"
+    ],
+    "image": null,
+    "rtr": false,
```
Comment (Cebion) [inline ports/crysalis/port.json:23]: Is this game r2p?

Comment (cdeletre) [reply ports/crysalis/port.json:23]: It's not rtr but you make me think that I've put license files (rpgmaker RTP and music) for stuffs that the user has to import by himself. I'll check with Laggy ;)

Comment (cdeletre) [reply ports/crysalis/port.json:23]: So, Laggy and I came to the conclusion that it won't be rtr (kind of hard to get in touch with the author of this 2011 game). I've removed the useless license files in commit 
1ca9e36

**Pull request merged**